docs/proton-kafka.md

```diff
@@ -106,233 +114,326 @@ annotations:
-If the external stream is used to write data to a Kafka topic and the `data_format` is set to `JSONEachRow`, setting this to `true` to make sure each Kafka message only contains one JSON document.
+
+Set to `true` to ensure each Kafka message maps to exactly **one JSON document**, especially when writing with `JSONEachRow`.
 
 #### kafka_schema_registry_url
-Set to the address of Kafka Schema Registry server. `http` or `https` need to be included. [Learn more](/proton-schema-registry) for this setting and other settings with `kafka_schema_` as the prefix.
+
+URL of the [Kafka Schema Registry](/proton-schema-registry), including the protocol is required (`http://` or `https://`).
 
 #### kafka_schema_registry_credentials
-Set in the 'username:password' format. [Learn more](/proton-schema-registry).
 
-#### ssl_ca_cert_file
-Set to the path of the CA certificate file.
+Credentials for the registry, in `username:password` format.
+
+#### ssl_ca_cert_file / ssl_ca_pem
 
-#### ssl_ca_pem
-Set to the content of the CA certificate file. Usually starts with `-----BEGIN CERTIFICATE-----`.
+Use either:
 
-#### skip_ssl_cert_check
-Default to `false`. Set to `true` to skip the SSL certificate check and don't specify the CA certificate file or content.
+* `ssl_ca_cert_file='/path/to/cert.pem'`
+* `ssl_ca_pem='-----BEGIN CERTIFICATE-----\n...'`
+
+#### `skip_ssl_cert_check`
+
+* Default: `false`
+* Set to `true` to **bypass SSL verification**.
 
 #### properties
 
-For more advanced use cases, you can specify customized properties while creating the external streams. Those properties will be passed to the underlying Kafka client, which is [librdkafka](https://github.com/confluentinc/librdkafka/blob/master/CONFIGURATION.md). Please refer to the [section](#advanced_settings) in the bottom of this page for more details.
+Used for advanced configurations. These settings are passed directly to the Kafka client ([librdkafka config options](https://github.com/confluentinc/librdkafka/blob/master/CONFIGURATION.md)) to fine tune the Kafka producer, consumer or topic behaviors.
+
+For more, see the [Advanced Settings](#advanced_settings) section.
+
+## Reading Data from Kafka
+
+Timeplus allows reading Kafka messages in multiple data formats, including:
 
-## Read Data in Kafka
-### Read messages in a single column {#single_col_read}
+* Plain string (raw)
+* CSV / TSV
+* JSON
+* Protobuf
+* Avro
 
-If the message in Kafka topic is in plain text format or JSON, you can create an external stream with only a `raw` column in `string` type.
+### Reading Kafka Messages as Raw String 
 
-Example:
+Use this mode when:
+
+* Messages contain **unstructured text or binary data**
+* No built-in format is applicable
+* You want to **debug raw Kafka messages**
+
+#### Example
 
 ```sql
 CREATE EXTERNAL STREAM ext_github_events
          (raw string)
 SETTINGS type='kafka',
          brokers='localhost:9092',
-         topic='github_events'
+         topic='application_logs'
```

Comment:
if you change the topic name, also need to change the ext stream name in line 197